src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
```

Comment:
```suggestion
    SECOND(5, true, 30_000_000),
    THIRD(5, false, 1_500_000),
```
`보너스일치여부`를 필드로 가지고있으면 `등수조건`을 더 명확히 파악할수 있지 않을까요?🤔

Reply:
오 네! 그렇게 수정해보겠습니다.